src/moscot/problems/time/_mixins.py

```diff
@@ -449,20 +448,19 @@ def prior_growth_rates(self: TemporalMixinProtocol[K, B]) -> Optional[pd.DataFra
     @property
     def posterior_growth_rates(self: TemporalMixinProtocol[K, B]) -> Optional[pd.DataFrame]:
         """Return the posterior estimate of growth rates of the cells in the source distribution."""
-        # TODO(michalk8): FIXME
         cols = ["posterior_growth_rates"]
         df_list = [
             pd.DataFrame(problem.posterior_growth_rates, index=problem.adata.obs.index, columns=cols)
             for problem in self.problems.values()
         ]
-        tup = list(self)[-1]
+        indices_remaining = set(self.adata.obs_names) - {ind for df in df_list for ind in df.index}
```

Comment:
This can be done in a nicer way: I'd first concatenate the `df_list` and then use a 2nd concat to add the remaining indices.